modules/aggregations/src/yamlRestTest/resources/rest-api-spec/test/aggregations/percentile_ranks_tdigest_metric.yml

```diff
@@ -70,21 +96,21 @@ filtered:
             percentile_ranks_int:
               percentile_ranks:
                 field: int
-                values: [50]
```

Comment:
Why the change from 50 to 51?

Reply:
The value for 50 is 0, then goes to 16 for 51. I'm fine with either 50-0 or 51-16, up to you.